engine/player/soulbinds.cpp

```diff
@@ -553,28 +572,36 @@ void combat_meditation( special_effect_t& effect )
   struct combat_meditation_buff_t : public stat_buff_t
```

Comment:
The actual changes are to this & the one change in sc_sim.hpp. Everything else was clang format. Oops.
I can revert and redo the changes without the accidental clang format if preferred.